problemsets-mat223/modules/module3-exercises.tex

```diff
@@ -1,10 +1,64 @@
 \begin{exercises}
+		% Topics:
+		% Spans
+		% Representing lines/planes/volumes as spans
+		% translated spans
+		% linear independence
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob
+		Find the spans of the following sets of vectors.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+				\item   $\Set*{\mat{1\\1}}$ % indep
+				\item   $\Set*{\mat{1\\3},\mat{2\\6}}$ % dep
+				\item   $\Set*{\mat{2\\4},\mat{4\\2}}$ % indep
+				\item   $\Set*{\mat{1\\2},\mat{-1\\-2}}$ % dep
+				\item   $\Set*{\mat{1\\2},\mat{-1\\2}}$ % indep
+				\item   $\Set*{}$
+				\item		$\Set*{\mat{1\\2},\mat{2\\3},\mat{3\\4}}$ % dep
+				\item   $\Set*{\mat{5\\4\\-3},\mat{1\\1\\0},\mat{2\\2\\2}}$ % indep
+				\item   $\Set*{\mat{1\\-2\\0},\mat{4\\-5\\1}}$ % indep
+				\item   $\Set*{\mat{1\\-2\\0},\mat{4\\-5\\1},\mat{5\\-7\\1}}$ % dep
 		\end{enumerate}
+		\prob
+		\begin{enumerate}
+			\item Determine whether the sets of vectors in question 1 are linearly
+			independent or dependent.
+
+			\item Is the set $\Set*{\mat{1\\2\\3\\4},\mat{5\\6\\7\\8},
+			\mat{9\\10\\11\\12}, \mat{13\\14\\15\\16}}$ linearly independent or dependent?
+
+			\item Are $n+1$ vectors in $\R^n$ linearly independent or dependent?
+		\end{enumerate}
+		\prob
+		\begin{enumerate}
+			\item
+			Determine which of the following lines and planes can be expressed as spans.
```

Comment:
We need to specify for each whether they are in R^2 or R^3. Let's enhance this by saying "express as  a span if possible, otherwise justify why it cannot be expressed as a span"